docs/set.md

```diff
@@ -35,3 +36,35 @@ odf set ceph log-level osd crush 20
 ```
 
 Once the logging efforts are complete, restore the systems to their default or to a level suitable for normal operations.
+
+### osd
+
+#### full-ratio
+
+This command allows updating the ceph osd full ratio in case ceph prevents the I/O operation on OSDs that reached the capacity specified.
+
+**Note**: If the value is set too close to 1.0, the cluster will be unrecoverable if the OSDs are full and there is nowhere to grow.
+
+``` bash
+odf set ceph full-ratio 0.9
```

Comment:
Now I'm actually wondering if we should have these as higher level settings instead of under ceph, let's discuss. There shouldn't be any conflict with other components setting a similar setting.
```suggestion
odf set full-ratio 0.9
```

Reply:
ass discussed moving it top-level command